src/Illuminate/Routing/Exceptions/InvalidSignatureException.php

```diff
@@ -15,4 +15,14 @@ public function __construct()
     {
         parent::__construct(403, 'Invalid signature.');
     }
+
+    public static function forInvalidSignature()
+    {
+        return new parent(403, 'Invalid signature.');
```

Comment:
This actually needs to be `new static` otherwise an instance of `HttpException` is returned instead of `InvalidSignatureException`.